<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMainWindow</class>
 <widget class="QMainWindow" name="FMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hacc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/hacc.qrc">
    <normaloff>:/items/money.png</normaloff>:/items/money.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBook">
    <property name="title">
     <string>Book</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionIcons"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMDI_mode"/>
   </widget>
   <addaction name="menuBook"/>
   <addaction name="menuView"/>
   <addaction name="menuManage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="actionsWindowsToolbar">
   <property name="windowTitle">
    <string>Windows toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="transactionsBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../res/hacc.qrc">
     <normaloff>:/main/icons/document-new.png</normaloff>:/main/icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../res/hacc.qrc">
     <normaloff>:/main/icons/document-open-data.png</normaloff>:/main/icons/document-open-data.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../res/hacc.qrc">
     <normaloff>:/main/icons/document-close.png</normaloff>:/main/icons/document-close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../res/hacc.qrc">
     <normaloff>:/main/icons/application-exit.png</normaloff>:/main/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionIcons">
   <property name="icon">
    <iconset resource="../../res/hacc.qrc">
     <normaloff>:/items/icon.png</normaloff>:/items/icon.png</iconset>
   </property>
   <property name="text">
    <string>Icons</string>
   </property>
  </action>
  <action name="actionMDI_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MDI mode</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="../../res/hacc.qrc">
     <normaloff>:/main/icons/dialog-ok.png</normaloff>:/main/icons/dialog-ok.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/hacc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>FMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
